operatorconfig/moduleconfig/common/version-values.yaml

```diff
@@ -24,3 +24,7 @@ powerflex:
   v2.6.0:
       authorization: "v1.6.0"
       observability: "v1.5.0"
+  v2.7.0:
```

Comment:
I think there may be a release of auth coming, not sure about observability, have you checked with those teams?